meta/README.md

```diff
@@ -0,0 +1,21 @@
+# meta
+
+Reference data used by the `pulumi-aws-native` provider.
+
+## ref-db.json
+
+Metadata describing the behavior of the [CloudFormation Ref Intrinsic](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-ref.html).
+
+What Ref returns varies from resource to resource. The only available information on this is free-form text in CloudFormation documentation and anything that can be extracted experimentally. To facilitate building tooling such as `pulumi/pulumi-cdk` that emulates CF behavior, `ref-db.json` collects schematized information describing the behavior of `Ref`.
+
+`ref-db.json` can be edited directly to correct errors or add entries for resources not yet covered.
```

Comment:
Can you add an example here on what it looks like to update this?

Reply:
You send a PR and edit it? I'm not sure I understand. It just has to conform to the schema.